tests/simple/simple.go

```diff
@@ -0,0 +1,186 @@
+package simple
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/cel-spec/tools/celrpc"
+
+	ctpb "github.com/google/cel-spec/proto/test/v1/conformanceTest"
+	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
+)
+
+var (
+	trueval = &exprpb.Value{ Kind: &exprpb.Value_BoolValue{true} }
+)
+
+// Match checks the expectation in the response matcher against the
+// actual result of evaluation.  Returns nil if the expectation
+// matches the actual, otherwise returns an error describing the difference.
+// Calling this function implies that the interpretation succeeded
+// in the parse and check phases.  See MatchValue() for the normalization
+// applied to values for matching.
+func Match(m *ctpb.EvalResponseMatcher, actual *exprpb.ExprValue) error {
+	switch m.Kind.(type) {
+	case *ctpb.EvalResponseMatcher_Value:
+		switch actual.Kind.(type) {
+		case *exprpb.ExprValue_Value:
+			return MatchValue(m.GetValue(), actual.GetValue())
+		}
+		return fmt.Errorf("Expected value, got %v", actual)
```

Comment:
Here and below, the test error message convention in Go is to say `Got xxx, want yyy`. See https://golang.org/doc/code.html#Testing

Reply:
Done.